src/test/java/net/snowflake/client/jdbc/SnowflakeDriverLatestIT.java

```diff
@@ -1451,7 +1451,10 @@ public void testUploadWithGCSPresignedUrlWithoutConnection() throws Throwable {
     File destFolder = tmpFolder.newFolder();
     String destFolderCanonicalPath = destFolder.getCanonicalPath();
     try {
-      connection = getConnection("gcpaccount");
+      Properties paramProperties = new Properties();
```

Comment:
Please revert the change in this test because this is supposed to be testing uploading the presigned URL without connection

Reply:
revverted